alibi_detect/cd/base_online.py

```diff
@@ -0,0 +1,303 @@
+from abc import abstractmethod
+import logging
+import numpy as np
+from typing import Callable, Dict,  Optional, Union
+from alibi_detect.base import BaseDetector, concept_drift_dict
+from alibi_detect.utils.frameworks import has_pytorch, has_tensorflow
+
+if has_pytorch:
+    import torch  # noqa F401
+
+if has_tensorflow:
+    import tensorflow as tf  # noqa F401
+
+logger = logging.getLogger(__name__)
+
+
+class BaseMMDDriftOnline(BaseDetector):
```

Comment:
It looks like except for the 2 kwargs and attributes `n_kernel_centers` and `lambda_rd_max`, the online MMD and LSDD base methods are exactly the same. Probably worth merging the base class then?